src/modules/colorPicker/ColorPickerUI/Controls/ColorFormatControl.xaml

```diff
@@ -173,7 +171,10 @@
                 Style="{StaticResource SubtleButtonStyle}"
                 ToolTipService.ToolTip="{x:Static p:Resources.Copy_to_clipboard}">
                 <Button.Content>
-                    <ui:SymbolIcon FontSize="20" Symbol="Copy20" />
+                    <TextBlock
+                        FontFamily="Segoe Fluent Icons, Segoe MDL2 Assets"
+                        FontSize="20"
```

Comment:
WPF-UI uses a different icon set. The previous icon here is 20 pixels based, but we don't need to keep that. Please try to see what the default size looks like, or use 12 or 16...